control-plane/connect-inject/webhook/container_init.go

```diff
@@ -165,7 +171,7 @@ func (w *MeshWebhook) containerInit(namespace corev1.Namespace, pod corev1.Pod,
 		},
 		Resources:    w.InitContainerResources,
 		VolumeMounts: volMounts,
-		Command:      []string{initContainerCommandInterpreter, "-ec", buf.String()},
+		Command:      command,
```

Comment:
Any specific reason behind this refactoring?

Reply:
no